### PR TITLE
Feat/ntrn 359 auction

### PR DESCRIPTION
**Pull request by ratik**

### CODE REVIEW

File: contracts/auction/src/contract.rs [hunk 1]
```diff
@@ -454,64 +291,87 @@ fn is_deposit_open(current_timestamp: u64, config: &Config) -> bool {
...
+            }],
+        });
+        res = res.add_message(transfer_msg);
     }
 
     // UPDATE STATE
-    state.total_ust_delegated -= amount;
-    user_info.ust_delegated -= amount;
+    state.total_usdc_deposited -= amount_usdc;
+    state.total_atom_deposited -= amount_atom;
```
Comment (pr0n00gler) [inline contracts/auction/src/contract.rs:363]: I don't see where you update `user_info.usdc_deposited` or `user_info.atom_deposited`

Comment (ratik) [reply contracts/auction/src/contract.rs:363]: https://github.com/neutron-org/neutron-tge-contracts/blob/08cfea21db6356ce992b1e2ab87303446917d7b9/contracts/auction/src/contract.rs#L184

Comment (pr0n00gler) [reply contracts/auction/src/contract.rs:363]: In the deposit handler I see it for sure, but not in the withdraw handler

Comment (ratik) [reply contracts/auction/src/contract.rs:363]: fixed

**Pull request merged**